packages/frameworks-astro/module.d.ts

```diff
@@ -0,0 +1,19 @@
+declare namespace App {
+  interface Locals {
+    session: import('@auth/core/types').Session | null
+  }
+}
+
+declare module 'auth:config' {
+  import type { AuthConfig } from '@auth/core'
+  interface FullAuthConfig extends AuthConfig {
+    basePath: string
+  }
```

Comment:
The core has this now, you don't need to add it as extra here.

```suggestion
  interface FullAuthConfig extends AuthConfig {}
```

Reply:
Yep, this can be removed entirely